src/Raven.Server/Documents/Subscriptions/Stats/SubscriptionConnectionPerformanceStats.cs

```diff
@@ -0,0 +1,34 @@
+// -----------------------------------------------------------------------
+//  <copyright file="SubscriptionConnectionPerformanceStats.cs" company="Hibernating Rhinos LTD">
+//      Copyright (c) Hibernating Rhinos LTD. All rights reserved.
+//  </copyright>
+// -----------------------------------------------------------------------
+
+using System;
+
+namespace Raven.Server.Documents.Subscriptions.Stats
+{
+    public class SubscriptionConnectionPerformanceStats
+    {        
+        public long ConnectionId { get; set; }
+        public long BatchCount { get; set; }
+
+        public string ClientUri { get; set; }
+
+        public string Exception { get; set; }
+
+        public DateTime Started { get; set; } // started is for when pending starts
```

Comment:
maybe we should refactor property names instead of putting comments? Additionally we talked to use scopes for that to have consistent approach.

Reply:
done